src/components/SingleDatePicker.jsx

```diff
@@ -277,6 +277,7 @@ export default class SingleDatePicker extends React.Component {
     };
 
     const onOutsideClick = !withFullScreenPortal ? this.onClearFocus : undefined;
+    const onInitialVisibleMonth = initialVisibleMonth ? initialVisibleMonth : () => (date || moment())
```

Comment:
I think it would be better to remove `initialVisibleMonth` from the `this.props` deconstruction and then change this line to:
```
const initialVisibleMonth = this.props.initialVisibleMonth || () => (date || moment());
```
`onInitialVisibleMonth` doesn't make very much sense to me as a variable name.

You're also missing a semicolon which is likely causing a lint error.